EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1044,7 +1044,7 @@ std::optional<std::string> V2::World::determineProvinceOwnership(const std::vect
 	}
 	if (winner.empty())
 		return std::nullopt;
-	return winner;
+	return std::move(winner);
```

Comment:
In general, you shouldn't do a std::move on return values, as the compiler optimizes that for you (indeed, its required to do so).